.env.example

```diff
@@ -1,6 +1,14 @@
-HOST=0.0.0.0
-PORT=1337
-APP_KEYS="toBeModified1,toBeModified2"
-API_TOKEN_SALT=tobemodified
-ADMIN_JWT_SECRET=tobemodified
-JWT_SECRET=tobemodified
+HOST=0.0.0.0 # Which host does this project run on?
+PORT=1337 # Which port does this project run on?
+APP_KEYS= # The secret keys used to sign session cookies, follows the format [key1, key2, ...]
+API_TOKEN_SALT= # Salt used to generate API tokens
+ADMIN_JWT_SECRET= # The secret used to sign the JWT for the Admin panel.
+TRANSFER_TOKEN_SALT= # The salt used to generate a transfer token. Transfer tokens are used to migrate data between Strapi instances.
+DATABASE_HOST=localhost # The host of the database
+DATABASE_PORT=5432 # The port of the database
+DATABASE_NAME=cow-cms-db # The name of the database
+DATABASE_USERNAME=cow-cms-user # The username of the database
+DATABASE_PASSWORD=cow-cms-db-password # The password of the database
+NODE_ENV=development # The environment the project is running in, development or production.
+DATABASE_CLIENT=postgres # The database client used to connect to the database, should be left postgres.
+DATABASE_SSL=false # Whether or not to use SSL when connecting to the database.
```

Comment:
This is great documentation, probably more than what we would expect, but great to have nonetheless.

One thing that would help and I imagine is what Anxo had in mind, is simply to split the env vars by category, like what we have in the Explorer https://github.com/cowprotocol/explorer/blob/1f6747358da9048061ea0b270e4dd274954a7779/.env.example#L1-L0 and CoW Swap https://github.com/cowprotocol/cowswap/blob/c1a5acf40f9d4d09af456e52ffbe0cc3dc6d5e54/.env.production#L1-L0

Reply:
Thank you, added.